cmd/agent/dist/conf.d/oracle-dbm.d/conf.yaml.default

```diff
@@ -0,0 +1,125 @@
+## All options defined here are available to all instances.
```

Comment:
What is the difference between this  and `pkg/collector/corechecks/oracle/config/conf.yaml.example` ?

Reply:
Deleted `pkg/collector/corechecks/oracle/config/conf.yaml.example` because is redundant. This file was created before adding [cmd/agent/dist/conf.d/oracle-dbm.d/conf.yaml.default](https://github.com/DataDog/datadog-agent/pull/16455/files/c9f55205fc33bf0c4d72f7a2e6405aa5ca013210#diff-b28fb23d10e9ff24e93485ad661398b26a960c3747929ac14981e1181fc757bd)